stories/Components/MegaMenu/megamenu.scss

```diff
@@ -0,0 +1,114 @@
+@import "../../assets/scss/variables";
+
+.mg-mega-topbar {
+  width: 100%;
+  background: $mg-color-blue-900;
+  display: flex;
+  align-items: flex-start;
+}
+
+.mg-mega-topbar__item {
+  padding: $mg-spacing-100 $mg-spacing-150;
+  color: $mg-color-white;
+  &:hover {
+    background: $mg-color-blue-800;
+  }
+}
+
+.mg-mega__button {
+  display: inline-block;
+  background-color: $mg-color-white;
+  padding: $mg-spacing-100 $mg-spacing-300;
+  border-radius: $mg-spacing-300;
+  border: 1px solid $mg-color-blue-900;
```

Comment:
We'll hit theming issues here. Suggest we make use of 

- `$mg-color-interactive`
- `$mg-color-interactive-active`

Won't theme everything, but will get us a long way.